### DISCUSSION TITLE
"The video is not being played."

### DISCUSSION THREAD

**Discussion by guilhermejuniorjuk** [pear-devs/pear-desktop#1628] [Q&A] [answered]
"Instead of the music video, another video appears, as if there is some sort of block. The music plays normally."

**Comment by inson1** [comment]
@guilhermejuniorjuk Is that reproducible? Do you have video?

**Reply by guilhermejuniorjuk** [reply]
A video that is not of the song is played

https://github.com/th-ch/youtube-music/assets/3765101/75d9e7ab-80b5-4bd1-8c55-11d0eb9a765a

**Reply by inson1** [reply, marked as answer]
@guilhermejuniorjuk that is plugin "Visualizer: Different music visualizers" with type butterchurn, just disable it by clicking on enabled (and then restart the ytb music)

and then please close this discussion :)

**Comment by guilhermejuniorjuk** [comment]
solved
